src/manifest.json

```diff
@@ -10,6 +10,7 @@
     "content_scripts": [
         {
             "matches": [
+                "*://lastfm-img2.akamaized.net/*",
```

Comment:
Because when getting the album art from Spotify, it is getting it from lastfm-img2 server, and without this, chrome.notifications API will fail to load album art and throw exception.